config/samples/raid-disks/eks-daemonset-raid-disks.yaml

```diff
@@ -29,12 +29,97 @@ spec:
             set -o errexit
             set -o nounset
             set -o pipefail
+            declare -A supported_oss
```

Comment:
Yes, I see that nvme is included in the docker startup script, but what I'm trying to automate is step 2c in the following document.
https://docs.nebula-graph.io/3.5.0-sc/k8s-operator/4.cluster-administration/4.4.storage-management/4.4.1.use-local-pv/

nvme-cli actually needs to be installed in the underlying node where eks-daemonset-raid-disks.yaml is run since it's being invoked by nsetter to run on the node in manage-startup-script.sh in the image. It will not use the nvme-cli installed on the image.
https://github.com/vesoft-inc/nebula-operator/blob/master/hack/startup-script/manage-startup-script.sh#L27
https://github.com/vesoft-inc/nebula-operator/blob/master/hack/startup-script/manage-startup-script.sh#L22

The node could be started using any amazon linux, centos/rhel or debian/ubuntu OS. Not installing nvme-cli on the node results in a `bash: line 83: nvme: command not found` error when running the start up script in eks-daemonset-raid-disks.yaml. This was also observed by Snap.
https://nebulagraphsupport.slack.com/archives/C04KT37P0CC/p1718390950814649?thread_ts=1718388563.558399&cid=C04KT37P0CC

Reply:
I see,  but GKE node not need nvme-cli,  I read the information from path '/dev/disk/by-id/google-local-nvme-ssd'.

Reply:
I see, that makes sense. Will remove the installation from gke.

Reply:
Fixed